diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/LineLength.kt

```diff
@@ -88,7 +96,8 @@ class LineLength : Rule("line-length") {
                     LONG_LINE.warnAndFix(configRules, emitWarn, isFixMode,
                             "max line length ${configuration.lineLength}, but was ${it.length}",
                             offset + node.startOffset) {
-                        fixError(newNode, configuration, node)
+                        positionByOffset =calculateLineColByOffset(node.treeParent.text)
```

Comment:
I think, this will be re-assigned on every `forEach` iteration. You could add condition on `!::positionByOffset.isInitialized`.